.claude/agents/moai/manager-lead.md

```
@@ -0,0 +1,153 @@
+```bash
+mkdir -p .moai/state/verify/$MOAI_SESSION_ID/
+go test -run TestX ./pkg 2>&1 | tee .moai/state/verify/$MOAI_SESSION_ID/M1.AC-XXX-001.log
+```
+
+The path MUST resolve at audit time (per `.claude/rules/moai/core/verification-claim-integrity.md` §2 baseline-attribution — a cited evidence path that no longer resolves is an unattributed claim). `.moai/state/verify/` is the canonical persistence location (NOT `/tmp`, which the OS clears). Any AC whose evidence could not be populated is marked `GAP` in Step 2 — never `PASS`.
+
+### Step 2 — Append fold row
+
+Append a row to `progress.md` §E.2 in the existing fold-row format:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)